### DISCUSSION TITLE
Use input variable for a certian parameter in xarray

### DISCUSSION THREAD

**Discussion by scottthomaswx** [pydata/xarray#5592] [Q&A]
So I am trying to automate a process where I plot meteorological data on a map from a NETCDF file. 
I attempted to create a process where I type in the parameter ID (e.g. cape_1000ml for 1000 mb mixed-layer CAPE), and it's passed to the line where I pull that parameter from the file. However, I keep getting an error saying the following: 

AttributeError: 'Dataset' object has no attribute 'parameter'

Is there a way I can do what I'm looking to do? The relevant parts of the code are below. Thanks,

```python
#Import Needed Packages

#Get Needed Inputs
filename = input('What is the name of the file: ')
time = input('Input your time in the following format YYYY-MM-DDTHH:MM:SS ')
parameter = input('What is the ID of the parameter you are plotting')
metunit = input('What are the units for the parameter you are plotting ')


#Open Dataset
ds = xr.open_dataset(filename)
ds

#Select Location, Parameter, And Date/Time

array1 = ds.parameter.loc[time, 41:37, 268:272]
array1.attrs['units']= metunit

...

#Plot Parameter
array1.plot.contourf(ax=ax,
                  levels = np.arange(0, 12, 0.5),
                  transform=ccrs.PlateCarree(), cbar_kwargs={'label': LAPSE24.units}, cmap = cmocean.cm.phase)

```

**Comment by keewis** [comment]
did you check the case of the parameter? With the information you gave I suspect you need to access `.Parameter` instead of `.parameter`.

If that's not right please add the `repr` of `ds` and the traceback of the error to the initial comment because otherwise I can't really tell what's going on.
